src/mixer/plugins/volume_mapping.c

```diff
@@ -111,9 +106,9 @@ static double get_normalized_volume(snd_mixer_elem_t *elem,
 	if (use_linear_dB_scale(min, max))
 		return (value - min) / (double)(max - min);
 
-	normalized = exp10((value - max) / 6000.0);
+	normalized = pow(10, (value - max) / 6000.0);
```

Comment:
Why that?

Reply:
There was a patch like this for alsa-utils that was accepted upstream. I figure it’s the same.

Reply:
That doesn't answer my question.

Reply:
exp10 is a GNU extension, not part of C++, and not available everywhere.

Reply:
Reworded commit message.

Reply:
Did so again.